channels_redis/pubsub.py

```diff
@@ -44,18 +57,6 @@ def __getattr__(self, name):
         else:
             return getattr(self._get_layer(), name)
 
-    def serialize(self, message):
-        """
-        Serializes message to a byte string.
-        """
-        return msgpack.packb(message)
-
-    def deserialize(self, message):
-        """
-        Deserializes from a byte string.
-        """
-        return msgpack.unpackb(message)
```

Comment:
If they were overridden and called with `super()` this may break.

I can keep these methods for backward compatibility and just remove the "assignment" in the `__init__` method.

Reply:
Yeah my preference is that we restore this, just for backwards compat reasons